events/2021.03.Online/TDs/TUM/HueIndoorSensor1.td.jsonld

```diff
@@ -0,0 +1,261 @@
+{
+    "@context": ["https://www.w3.org/2019/wot/td/v1", {
+        "@language": "en"
+    }],
+    "title": "Hue Indoor Sensor 1",
+    "id": "urn:dev:ops:32473-HueIndoorSensor-1",
+    "securityDefinitions": {
+        "basic_sc": {
+            "scheme": "basic",
+            "in":"header"
+        }
+    },
```

Comment:
`security` property is missing (`"security": "basic_sc"` etc.).
The same applies to `HueIndoorSensor2.td.jsonld`

Reply:
Fixed now!